app/controllers/spina/registers_controller.rb

```diff
@@ -31,9 +31,9 @@ def show
 
       records =
         case params[:status]
-        when 'closed'
+        when 'archived'
```

Comment:
I think this is OK for now, as this is private beta, but in future we should refrain from changing these params even when the UI changes, as it will break bookmarked URLs.

Reply:
+1